docs/DeveloperGuide.md

```diff
@@ -312,49 +307,69 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+**Use case: UC02 - Delete a profile**
+
+**MSS**
+
+1. User requests to <u>view all profiles (UC01)</u>.
+2. User requests to delete a specific profile in the list.
+3. NUScheduler deletes the person.
 
-    * 3a1. NUScheduler shows an error message.
+    Use case ends.
 
-      Use case resumes at step 2.
+**Extensions**
+
+* 2a. The given index is invalid.
+
+    * 2a1. NUScheduler shows an error message.
+
+      Use case ends.
 
-**Use case: Add an event**
+**Use case: UC03 - Add an event**
 
 **MSS**
 
-1. User requests to add an event to the schedule
-2. NUScheduler adds the event and displays the updated schedule of events
+1. User requests to add an event to the schedule.
+2. NUScheduler adds the event and displays the updated schedule of events.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. The starting date or the ending date is not provided.
+* 1a. The Title, starting date or the ending date is not provided.
```

Comment:
Not sure if we are simplifying this, but if we are not, I think `name` might be a better word to match our User Guide.